pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,313 @@
+
+
+Slack Server (Socket Mode)->>my-slack-bot: Sending "reaction_added" payload
+
+
+Note left of my-slack-bot: Event received
+``` 
+
+## Background context
+[WebSocket](https://en.wikipedia.org/wiki/WebSocket) is a communication protocol that enables full-duplex, bidirectional data exchange between a client and a server over a single, long-lived connection. Unlike HTTP, which relies on the request-response model, WebSocket is ideal for scenarios where real-time, interactive and low-latency communication is necessary.
+
+
+```mermaid
+sequenceDiagram
+Client -->> Server: Initiate WebSocket Connection
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+Server -->> Client: WebSocket Response
+```
+
+In Slack, WebSocket is employed as part of its [Socket Mode](https://api.slack.com/apis/connections/socket) feature to facilitate real-time notifications between Slack's servers and third-party applications or bots.  Socket Mode simplifies Slack app development by eliminating complex infrastructure challenges faced with HTTP, offering simpler and more feature driven solutions for app developers.
+
+
+The [Slack Event API](https://api.slack.com/apis/connections/events-api) is a tool that lets developers receive real-time notifications of specific events in a Slack workspace. By subscribing to event types like messages, reactions, and user presence changes, this API enhances automation within the Slack platform, making it a powerful resource for building custom integrations and chatbots.
+
+
+
+## Define AsyncAPI Version, API Information, and Server
+
+You start your AsyncAPI document by specifying the AsyncAPI version and essential information about your Slack application's API which includes details such as the `title`, `version` and `description`.
+
+The `ws` server section allows you to define the protocol and specify information about the URLs your application will use, such as  `host`, `pathname`, and `description`.
+
+The WebSocket URL is generated  by invoking the [apps.connections.open](https://api.slack.com/methods/apps.connections.open) method from Slack’s API. You use the authorization tokens obtained during the configuration of  my-slack-bot to generate this URL.
+
+```
+asyncapi: '3.0.0'
+info:
+  title: MySlackBot API
+  version: '1.0.0'
+  description:  |
+    The MySlackBot App manages popular messages in a workspace by monitoring message reaction data from Slack's Event API.
+    
+servers:
+  ws:
+    host: wss://wss-primary.slack.com/
+    pathname: link/?ticket=13748dac-b866-4ea7-b98e-4fb7895c0a7f&app_id=fe684dfa62159c6ac646beeac31c8f4ef415e4f39c626c2dbd1530e3a690892f
+    protocol: wss
+    description: "Websocket URL generated to communicate with Slack"
+```
+
+
+## Define Operations and Channels
+The  `operation` property, is all about defining specific tasks your application can perform. Essentially, it's how my-slack-bot interacts with Slack.
```

Comment:
```suggestion
The  `operation` property, is all about defining specific tasks your application can perform. Essentially, it's how `my-slack-bot` interacts with Slack.
```

always when you mention `my-slack-bot` in this document, please make it with back tick so it renders as code,